m4_sessions/1808-inning/surprise_of_happiness.md

```diff
@@ -24,9 +36,17 @@
      All sessions NEED to have a closing activity, reflection, discussion
     
 ## Introductons 
+
+     name, mod, and "What do you think makes you happy?"
```

Comment:
Maybe write this on the board so you have tons of happy things on the board.

Reply:
Love this idea!